### ISSUE TITLE
trivial questions and suggestions

### ISSUE THREAD

**Issue by qiulesun** [zhanghang1989/PyTorch-Encoding#179]
It is an awesome and developing repo., containing more SOTA methods.
Question 1
For obtaining mIoU with backgroud , computing mIoU with backgroud, i,e,. 60 classes for PContext, fisrt need to get mIoU without backgroud (mIoU_59) as you did in repo, then mIoU_60 is directly equal to mIoU_59 * 59 / 60 and will be slightly weaker than mIoU_59. Do I understand correctly?
Question 2
For multisize evaluation, will you consider employing dense crop on feature map rather than on input image? It drastically reduces computational overhead and may further boost the performance.
Question 3
Do you consdier the usage of accumulation gradient strategy to update param. due to limited GPU memory (small batchsize)?
Question 4
Your work appeals to me. When will you release your cvpr2019 paper Co-occurrent Features in Semantic Segmentation ?

Thank you for your consideration and I am looking forward to your reply.

**Comment by zhanghang1989** [comment]
Q1: yes, we just simply assume the pixels from background category are all misclassied.
Q2: I agree. But for some model you may need to seperate the backbone and "head" to apply that trick, as they may not be fully convolutional (or change behavior when image size changes).
Q3: That won't work. Please see the blog for how batch normalization works https://hangzhang.org/blog/SynchronizeBN/
Q4: Will do. Currently blocked by the Amazon legal apprvoal.

Thanks!

**Comment by qiulesun** [comment]
**Requirement 1**
Will you consider releasing both results and codes of usage of Cityscapes using your repo.?

**Comment by qiulesun** [comment]
@zhanghang1989  [Performance on Pascal Context #78]( https://github.com/HRNet/HRNet-Semantic-Segmentation/issues/78)

**Comment by qiulesun** [comment]
@zhanghang1989  I note that there are two different ways to compute mIoU on 60 classes for PASCAL Context. One is used like EncNet (mIoU_60 = mIoU_59 * 59 / 60.), another is from issue [Performance on Pascal Context #78](https://github.com/HRNet/HRNet-Semantic-Segmentation/issues/78). Which one is more reasonable ？

**Comment by zhanghang1989** [comment]
I agree with the comment https://github.com/HRNet/HRNet-Semantic-Segmentation/issues/78#issuecomment-571777824

Giving up predicting the back ground class is actually easier for the network. Depending on how you want to tackle this problem, you may choose a consistent way to compare with other work. Either way is okay, as long as the comparisons are fair.
